lib/smack/BoogieAst.cpp

```diff
@@ -29,7 +29,7 @@ const Expr *Expr::or_(const Expr *l, const Expr *r) {
 }
 
 const Expr *Expr::cond(const Expr *c, const Expr *t, const Expr *e) {
-  return new CondExpr(c, t, e);
+  return if_then_else(c, t, e);
```

Comment:
Rename `if_then_else` into `ifThenElse`.

Reply:
I also think we should rename this `cond` generator into `ifThenElse` to make it consistent.

Reply:
I removed `cond` in favor of `ifThenElse`. They are semantically equivalent now, and there's no reason to keep both around.